treeopt/modules/simulate.py

```diff
@@ -0,0 +1,9 @@
+def simulateBenchmarkFunction(fun, x):
```

Comment:
This seems a bit overengineered. Is this really needed? Wouldn't it be easier to simply evaluate the function where ever it is needed?

Reply:
Ok, I still think this is a bit too much indirection and overengineered, but I suppose it doesn't hurt